src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -317,7 +317,7 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 files.AddRange(srcElement.Value.Trim(';').Split(';').Select(s => 
                     new ManifestFile
                     {
-                        Source = s.SafeTrim().TrimStart(slashes),
```

Comment:
I am concerned about removing this functionality. When did this functionality exist? Was it provided in the old forked version of `dotnet pack`? Is it available in `3.3.0`?